packages/components/tests/dummy/app/templates/components/power-select.hbs

```diff
@@ -0,0 +1,107 @@
+{{page-title "PowerSelect Component"}}
+
+<h2 class="dummy-h2">PowerSelect</h2>
+
+<section>
+  <h3 class="dummy-h3" id="overview"><a href="#overview" class="dummy-link-section">§</a> Overview</h3>
```

Comment:
Probably I would expand this overview, explaining that is not an HDS component but just the styling (we're essentially "re-skinning" it to make is more similar to the other form controls).